Fix smoke test

ch1bo opened the pull request:
* Fixes the `exes` nix shell so we can run the smoke test again

* Reverts the wait times back to 600 seconds for the smoke test scenario

* Took note of a recently published `hydra-scripts-tx-id`

* Fixed also the contestation period to a higher value, now 100 seconds instead of 1


Red bin items discovered
- Should parameterize the `Scenarios` by network they run in, i.e. the block production time is a parameter for a good contestation period + max wait time for an end-to-end scenario
- Hydra node state was persisted and re-used by the smoke test, even after we changed the contestation period -> This was a surprise when investigating

ch1bo committed:
Use ?submodules=1 to access exes shell

This is required because the hydra-cluster package is included in this
shell environment and it requires submodules to compile.

ch1bo committed:
Revert back to longer waits in smoke test scenario

We really need to make these scenarios fully network-agnostic..

ch1bo committed:
Add a txid for preview

v0d1ch reviewed (approved):
Changes look good to me, I'll spin it locally to see if I can reproduce.

v0d1ch committed:
Alter the contestation period to 100

- This scenario also needs to run on a public testnet where
block production is usually 20+ seconds. So one second is too low value
for a contestation period especially since we use it for calculating the
validity range of close tx.

v0d1ch committed:
Cleanup hydra-node state in smoke-tests

- The smoke tests should clear the hydra-node state on each run so that
we do not continue using previous head state.

v0d1ch committed:
Increase the wait time for HeadIsFinalized

ch1bo commented:
Latest run was green :green_heart: https://github.com/input-output-hk/hydra/actions/runs/4291889526.

Thanks @v0d1ch for fixing this, I would approve if I could :)
